test/test-spawn.c

```diff
@@ -1913,9 +1913,9 @@ TEST_IMPL(spawn_exercise_sigchld_issue) {
   options2.args = args;
   options2.exit_cb = fail_cb;
   options2.flags = 0;
+  uv_process_t process[100];
   for (i = 0; i < 100; i++) {
-    uv_process_t process;
-    r = uv_spawn(uv_default_loop(), &process, &options2);
+    r = uv_spawn(uv_default_loop(), &process[i], &options2);
     ASSERT(r == UV_ENOENT || r == UV_EACCES);
```

Comment:
why was ASAN unhappy with this? I thought that if `uv_spawn` returns non-zero that the memory could be immediately freed and reused

Reply:
your guess is as good as mine 🤷🏻  you can see the logs here: https://github.com/nornagon/libuv/runs/4959706174?check_suite_focus=true#step:6:797

Reply:
ah, nevermind, we do still need to call `uv_close` on it eventually (to make valgrind happy?)